README.md

```diff
@@ -4,7 +4,15 @@ Implement passwordless sign-in to your app using Solana wallets (Phantom, sollet
 
 This project has been implemented following best practices for sign-in security (following the OAuth spec) and user experience. It implements persistent sessions using refresh and access tokens, so your customers don't need to sign-in with their wallet each time they visit your site.
 
-It uses Firebase Auth for persistent sessions and user DB as it's free and battle tested by millions of apps, but it's built in a modular way so other auth backends coudl be plugged in. 
+It uses Firebase Auth for persistent sessions and user DB as it's free and battle tested by millions of apps, but it's built in a modular way so other auth backends coudl be plugged in.
 
-**Project status**: this project is functionally complete but not yet nicely packaged with NPM and so on and is lacking docs. 
+**Project status**: this project is functionally complete but not yet nicely packaged with NPM and so on and is lacking docs.
 We estimate to have that complete by Jan 23rd 2022
+
+### Configure domain
+
+You will need to define a environment variable to use when checking the payload send from the client. This has to be the same that is passed to `SolanaAuthProvider` on the client.
+
+```sh
+export SOLANA_AUTH_DOMAIN=example.xyz
```

Comment:
I think this domain will be the same no matter whether it's solana or ethereum auth or whatever

So I would call it AUTH_DOMAIN